dhis-2/dhis-api/src/main/java/org/hisp/dhis/common/ValueType.java

```diff
@@ -75,7 +75,13 @@
     URL( String.class, false ),
     FILE_RESOURCE( String.class, true, FileTypeValueOptions.class ),
     IMAGE( String.class, false, FileTypeValueOptions.class ),
-    GEOJSON( GeoJSON.class, false );
+    GEOJSON( GeoJSON.class, false ),
+    MULTI_TEXT( String.class, true );
+
+    public static List<String> splitMultiText( String value )
+    {
+        return value == null ? List.of() : List.of( value.split( "," ) );
```

Comment:
Question: We use `,` as separator which is a fairly commonly used character. Do we want to use `|` or some less used character as separator?

Reply:
The `,` was chosen as it should still be a unusual character to use as part of a code but it is very readable when displayed as string for a data value. I think we should assume that we will not be able to adjust the UI in every view to deal with multi-text data values and display them nicely. So whatever separator we chose should render "human readable". Comma was the obvious choice and in regards to usage as actual character of a code is no worse or better than alternatives like `;`, `:` or `+`.